_posts/2021-04-02-gs_00001_computer_setup.md

```diff
@@ -0,0 +1,453 @@
+suppliers. Please do not understand it as an explicit recommendation. It is more
+an example to show you that it is not necessary to have a super high-end
+computer. Of course, you can spend more if you like, but you can also downgrade
+a bit and get even cheaper. For things we plan to do in this blog, a computer as
+shown in the table will be sufficient.
+
+# Windows, Linux, Mac?
+
+So far we talked only about the hardware and not the software running on it. The
+main software on a computer is the operating system. It has control of your
+system and manages all your programs such as your browser, your email client or
+your music application. Windows is probably known by everyone, but there are
+also other operating systems available which got more and more attention over
+the past years. How do they differ? Which one is the best?
+
+**Microsoft Windows** is installed on most machines. Many people like to use it
+because it is very simple and intuitive. It does not require any IT knowledge. A
+huge advantage is that most software is available for Windows. Especially for
+gaming Windows should be your choice. Also, the office packet with Word, Excel,
+Teams and so on is very powerful. This makes it very interesting for companies
+as well. Consequently, most office computers run Windows. On the downside,
+Windows is not free to use. You need to buy a license to use it legally. Since
+Windows is on so many machines and its users often do not have IT knowledge it
+is the target of many computer viruses. This is reinforced by its complexity
+which leads space for security issues. Therefore, do not click on dubious links
+or download things from untrusted websites (this applies to all systems
+though!).
+
+**Mac OS** is an operating system developed by Apple. It is shipped with all
+Apple computers and intended to be used with Apple products only. So, it is not
+an option if you want to build your own computer. There are ways to install it
+on other machines, but this can be very tricky. Mac is different from Windows,
+but still very easy and intuitive to use. In my opinion, it runs more smoothly
+than Windows and looks nicer. While Windows sometimes hangs up, I never
+experienced this on a Mac. A Linux-like terminal (we come to that soon) lets
+programmers work efficiently. The disadvantage is the price of Apple products
+and the limited choice of products.
+
+**Linux** is a family of open-source operating systems. There are many so-called
+Linux distributions available which all are build on top of the Linux kernel,
+which is something like the heart of Linux. The kernel is combined with
+different software like a graphical interface to build a _distribution_. So,
+internally all distributions work the same, but they appear differently to the
+user. If you talk about Linux, you usually talk about all kinds of
+distributions. Linux is very popular among software developers. It is free to
```

Comment:
What about hardware developers?

Reply:
I did not put it in here, because there are many hardware tools that only exist for Windows or they are equal for both systems. Maybe I can just use developers in leave "software" out?